library-app/src/components/Books/BooksList/BooksList.tsx

```diff
@@ -1,18 +1,62 @@
 import BooksItem from '../BooksItem/BooksItem'
 import styles from './BooksList.module.css'
+import { useEffect, useState } from 'react'
+import Modal from '../../Layout/Modal'
+import ManageBookForm from './ManageBookForm'
+import { getBooksRequest } from '../../../services/BooksServices'
+import Author from '../../../models/author.model'
+
+export interface BookBodyDataGet {
+  Id: number
+  Title: string
+  Description: string
+  Isbn: string
+  Quantity: number
+  Cover: string
+  PublishDate: string
+  Authors: Author[]
+}
 
 const BooksList = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [books, setBooks] = useState<BookBodyDataGet[]>([
```

Comment:
Why is the initial state this book array with one hardcoded element?

Reply:
removed